shell/integration-test-setup.sh

```diff
@@ -4,12 +4,18 @@ set -e -o pipefail
 
 if [[ `uname -m` != "x86_64" ]]
 then
-  sudo apt install -y --no-install-recommends docker-compose
-  source bin/arm64_env.sh
+  sed -e 's/export //g; 1d;/\$/d' bin/arm64_env.sh > arm64.env
+  export ENV_FILE=arm64.env
+  export DOCKER_COMPOSE="nexus3.edgexfoundry.org:10003/edgex-devops/edgex-compose-arm64:latest"
 else
-  sudo curl -L https://github.com/docker/compose/releases/download/1.21.2/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
-  sudo chmod +x /usr/local/bin/docker-compose
-  source bin/env.sh
+  sed -e 's/export //g; 1d;/\$/d' bin/env.sh > x86_64.env
+  export ENV_FILE=x86_64.env
+  export DOCKER_COMPOSE="nexus3.edgexfoundry.org:10003/edgex-devops/edgex-compose:latest"
 fi
-bash deploy-edgeX.sh
+
+[ -f "$PWD/docker-compose-test-tools.yml" ] && export docker_compose_test_tools=$PWD/docker-compose-test-tools.yml
```

Comment:
Does this need to be an "if" line, or is it meant to just test and either succeed or fail? (This might be a good thing to add a comment to, since even though it's valid, it's a little unusual)

Reply:
You're right. The if statement is unnecessary. 
Removed.